EXERCISE_02/app/src/main/java/com/example/exercise_02/Data.java

```diff
@@ -0,0 +1,12 @@
+package com.example.exercise_02;
+
+public class Data {
```

Comment:
It is a good practice to set class names to something that can infer the usage of this class.
A good example could have been "Character" or "Person"